src/components/MapBox.vue

```diff
@@ -129,12 +136,12 @@ export default {
 
       let map = event.map; // This gives us access to the map as an object but only after the map has loaded
 
-      //pinch to zoom for touch devicesgit pull 
```

Comment:
lol

Reply:
You don't like my I thought I was typing in my terminal comment Aaron?